src/com/projectkorra/projectkorra/airbending/combo/AirSweep.java

```diff
@@ -62,6 +63,7 @@ public AirSweep(final Player player) {
 		this.speed = getConfig().getDouble("Abilities.Air.AirSweep.Speed");
 		this.knockback = getConfig().getDouble("Abilities.Air.AirSweep.Knockback");
 		this.cooldown = getConfig().getLong("Abilities.Air.AirSweep.Cooldown");
+		this.hitRadius = getConfig().getDouble("Abilities.Air.AirSweep.Radius");
```

Comment:
You didn't change the variable name here, your build won't compile otherwise.

Reply:
Sorry about this. I should've changed it in Eclipse rather than editing here.